presentation-compiler/src/main/dotty/tools/pc/completions/Completions.scala

```diff
@@ -801,7 +819,7 @@ class Completions(
         case _ =>
 
       // symbols whose owner is a base class are less relevant
-      if sym.owner == defn.AnyClass || sym.owner == defn.ObjectClass
+      if isInheritedFromScalaLibrary(sym)
       then relevance |= IsInheritedBaseMethod
```

Comment:
Perhaps
```
      // symbols which are universally available are less relevant
      if sym.owner == defn.AnyClass
      || sym.owner == defn.ObjectClass
      || sym.ownersIterator.contains(defn.ScalaPredefModuleClass)
      then relevance |= IsInheritedBaseMethod
```
plus the other special owner checks. Product and Equals?

Reply:
I don't think we want to penalize everything from `scala.Predef`. That is why I only selected things that get suggested everywhere - implicit conversions and methods inherited in case classes. For example, `Set[A](elems: A*)` is quite useful to have.

Reply:
I don't have an opinion about that and leave it to you all; I'd suggest a test (not exhaustive) that demonstrates priority of a useful thing, i.e., a test that my code would fail. (I promise that I will not, in future, having forgotten this conversation, PR my code.)